src/components/WalletCongresSenate/Motion/index.js

```diff
@@ -58,7 +61,7 @@ export default function Motion({
               <b>
                 {voting.nays.length}
                 /
-                {threshold}
+                {(membersCount / 2).toFixed()}
```

Comment:
Doesn't make sense. Why /2?

Reply:
If anything this should be `membersCount-threshold+1`, but TBH I expect this to confuse the hell out of people.